### ISSUE TITLE
Match movies - alot of equal - uses same match

### ISSUE THREAD

**Issue by KennyVB** [tidusjar/Ombi#2685]
#### Ombi build Version:

V 3.0.3988

#### Update Branch:

Open Beta

#### Media Sever:

Plex Media Server 

#### Media Server Version:

1.14.0.5470

#### Operating System:

Ubuntu 18LTS

#### Ombi Applicable Logs (from `/logs/` directory or the Admin page):

```

(Logs go here. Don't remove the ' tags for showing your logs correctly. Please make sure you remove any personal information from the logs)

```

#### Problem Description:

i have alot of different Robin Hood movies, 
when i search the new Robin hood movie (2018) it says it's allrdy there, alltho i have alot of other versions of roboin hood with different actors etc. not that one, and when i click (watch on plex) it goes to the front page because yes, it's not there!

#### Reproduction Steps:

add alot of different robin hood movies, matchs them correctly, then try search for a different one, all of them are now avalible on server even tho you dont have all of them

**Comment by goldenpipes** [comment]
this seems to work for me can you run a search in chrome incognito? thanks

**Comment by KennyVB** [comment]
hmm still same result.
it writes "Robin Hood 2018" sdp avalible and then plex link takes me to server not the movie because it dont have it..

what's sdp ?

**Comment by tidusjar** [comment]
No, it's the holiday season, I've not had much time recently as I'm spending time with family. 

Can you delete your ombi external.db and then restart ombi. It will resync with Plex then

**Comment by KennyVB** [comment]
Allrigth :) sorry about that!
just deleted external and it's running a scan (i think) so far so good :)
